### ISSUE TITLE
tvOS10 and xcode 8 seems to have issue

### ISSUE THREAD

**Issue by hsb1007** [jasarien/Provenance#423]
I am trying to set Provenance up and running on my apple tv with tvOS10.

It would not get compiled due to UIKit framework's UISelectionFeedbackGenerator.

I have removed "UIKIT_CLASS_AVAILABLE_IOS_ONLY(10_0)" and starting to throw below

Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_UISelectionFeedbackGenerator", referenced from:
      objc-class-ref in PVControllerViewController.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

If I don't remove those line, I get 

UISelectionFeedbackGenerator is unavailable: not available on tvOS

Does anyone have the same issue?

**Comment by jasarien** [comment]
Are you using the latest source? This issue has been fixed.

**Comment by hsb1007** [comment]
Oh let me try again then. I have re pulled from master though

**Comment by jasarien** [comment]
Also you said you removed `"UIKIT_CLASS_AVAILABLE_IOS_ONLY(10_0)` -- you removed this from the UIKit classes header? Don't do that. Never noticed the SDK files, you'll break everything.

**Comment by hsb1007** [comment]
YUP =) it's working fine now =) thank you very much!
